tutorials/beginner/index.md

```diff
@@ -22,34 +22,34 @@ beginner_sql
 
 ## [Fugue Interface](interface.ipynb)
-In this section, we'll start covering some concepts like the Directed Acyclic Graph (DAG) and the need for explicit schema in a distributed compute environment. We'll show how to pass parameters to `Transformers`, as well as load and save data. With these, users will be able to start some basic work on data through Fugue.
+This section will cover some concepts like the Directed Acyclic Graph (DAG) and the need for explicit schema in a distributed computing environment. We'll show how to pass parameters to `Transformers`, as well as load and save data. With these, users will be able to start some basic work on data through Fugue.
 
 ## [Joining Data](joins.ipynb)
-Here we'll show the different ways to join DataFrames in Fugue along with union, intersect, and except. SQL and Pandas also have some inconsistencies users have to be aware of when joining. Fugue maintains consistency with SQL (and Spark).
+Here we'll show the different ways to join DataFrames in Fugue along with union, intersect, and except. SQL and pandas also have some inconsistencies that users should be aware of when joining. Fugue maintains consistency with SQL (and Spark).
 
 ## [Extensions](beginner_extension.ipynb)
-We already covered the `transformer`, the most commonly used Fugue extension. Extensions are Fugue operations on `DataFrames` that are used inside the DAG. Here we will cover the `creator`, `processor`, `cotransformer` and `outputter`.
+We already covered the `transformer`, the most commonly used Fugue extension. Extensions are Fugue operations on `DataFrames` that are used inside the DAG. Here we will cover the `creator`, `processor`, `cotransformer`, and `outputter`.
 
-## [Distributed Compute](distributed_compute.ipynb)
-The heart of Fugue is distributed compute. In this section we'll show the keywords and concepts that allow Fugue to fully utilize the power of distributed compute. This includes `partitions`, `persisting`, and `broadcasting`.
+## [Distributed Computing](distributed_compute.ipynb)
+The heart of Fugue is distributed computing. In this section, we'll show the keywords and concepts that allow Fugue to fully utilize the power of distributed computing. This includes `partitions`, `persisting`, and `broadcasting`.
 
 ## [FugueSQL](beginner_sql.ipynb)
-We'll show a bit of [FugueSQL](../fugue_sql/index.md), the SQL interface for using Fugue. This is targeted for heavy SQL users and SQL-lovers who want to use SQL on top of Spark and Dask, or even pandas. FugueSQL that is used on DataFrames in memory as opposed to data in databases.
+We'll show a bit of [FugueSQL](../fugue_sql/index.md), the SQL interface for using Fugue. This is targeted at heavy SQL users and SQL-lovers who want to use SQL in addition to Spark and Dask, or even pandas. FugueSQL is used on DataFrames in memory as opposed to data in databases.
```

Comment:
Python and SQL are the interface and Spark and Dask are the backends so "on top of" is technically more right than "in addition", but happy to change the wording to make that clearer too.